thrall/app/controllers/HealthCheck.scala

```diff
@@ -4,18 +4,17 @@ import com.gu.mediaservice.lib.argo.ArgoHelpers
 import com.gu.mediaservice.lib.management.ElasticSearchHealthCheck
 import lib._
 import lib.elasticsearch._
-import lib.kinesis.ThrallMessageConsumer
 import play.api.Logger
 import play.api.mvc._
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, Future}
 
-class HealthCheck(elasticsearch: ElasticSearch, messageConsumer: ThrallMessageConsumer, config: ThrallConfig, override val controllerComponents: ControllerComponents)(implicit override val ec: ExecutionContext)
+class HealthCheck(elasticsearch: ElasticSearch, streamRunning: Future[_], config: ThrallConfig, override val controllerComponents: ControllerComponents)(implicit override val ec: ExecutionContext)
```

Comment:
This type signature looks really odd 🤔

Reply:
Is `=> Boolean` better? Leave the implementation to the wiring?